src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -77,8 +77,8 @@ public sealed partial class Font : MarshalByRefObject, ICloneable, IDisposable,
         /// Gets the face name of this <see cref='Font'/> .
         /// </summary>
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-#if !NETCORE
         [Editor ("System.Drawing.Design.FontNameEditor, " + Consts.AssemblySystem_Drawing_Design, typeof (System.Drawing.Design.UITypeEditor))]
+#if !NETCORE
         [TypeConverter (typeof (FontConverter.FontNameConverter))]
```

Comment:
I wonder if TypeConverter has a similar problem here?  It's another case where the ComponentModel infrastructure has a hard-coded table of converters by type name.